FinniversKit/Sources/Fullscreen/FrontPage/FrontPageView.swift

```diff
@@ -133,6 +147,20 @@ public final class FrontPageView: UIView, BasicFrontPageView {
     public func scrollToTop() {
         adRecommendationsGridView.scrollToTop()
     }
+    
+    public func changeCompactMarketsViewVisibilityStatus(to status: CompactMarketsViewVisibilityStatus) {
+        switch status {
+        case .displaying(progress: let progress):
+            compactMarketsView.isHidden = false
+            let height = compactMarketsView.calculateSize(constrainedTo: frame.width).height
+            compactMarketsViewBottomConstraint.constant = height * progress
+            layoutIfNeeded()
+        case .displayed:
+            compactMarketsView.isHidden = false
```

Comment:
This assumes you have gotten a `.displaying` with progress 1.0 before right? But I guess it should be impossible to go from, for instance, progress 0.5 directly to `.displayed` 😄